### PR TITLE
Fix some fr dot errors

### PR DESCRIPTION
**Pull request by lucasmichot**
Also ignore sass-cache

### CODE REVIEW

File: _data/fr.yml [hunk 1]
```diff
@@ -243,16 +243,16 @@ quality:
...
-        - 'Contenir des tests unitaires. Le but étant d’avoir une bibliothèque testée à plus de 80% pour la version 1.0 ;'
-        - 'Documenter tous les élements du code source avec des <a href="http://www.phpdoc.org/docs/latest/references/phpdoc/index.html" target="_blank">DocBlock</a> ;'
-        - 'Utiliser <a href="http://semver.org/lang/fr/" target="_blank">la gestion sémantique de version</a> pour gérer les mises à jour ;'
-        - 'Maintenir un fichier <a href="https://keepchangelog.com" target="_blank">CHANGELOG</a>'
-        - 'Utiliser <a href="https://travis-ci.org/" target="_blank">Travis-CI</a> pour continuellement vérifier les normes d’écriture et lancer ses tests unitaires ;'
+        - 'Utiliser <code>League</code> comme espace de nom pour la norme <a href="http://www.php-fig.org/psr/psr-4/">PSR-4</a>. Le code source est entièrement contenu dans le dossier <code>src</code>.'
+        - 'Adhérer à la norme <a href="http://www.php-fig.org/psr/psr-2/fr" target="_blank">PSR-2</a> comme style d’écriture de code.'
+        - '&Ecirc;tre reprise sur Packagist avec <code>league</code> comme espace de nom vendeur.'
```
Comment (GrahamCampbell) [inline _data/fr.yml]: Not correct. French grammar has spaces before `;` but not before `.`.

Comment (lucasmichot) [inline _data/fr.yml]: I don't see space before this dot...

Comment (lucasmichot) [inline _data/fr.yml]: @GrahamCampbell ?

Comment (GrahamCampbell) [inline _data/fr.yml]: Ohhh. lol - google chrome translated the diff...

Comment (reinink) [inline _data/fr.yml]: Haha, I had the same problem! :)

I really need to just block Chrome translating from GitHub entirely.

**Comment by lucasmichot** [comment]
google-translate-lady-gaga-error

**Pull request merged**